src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder result = new StringBuilder();
+        int[] salaries = new int[names.length];
+        SimpleDateFormat dateFormat = new SimpleDateFormat("dd.MM.yyyy");
```

Comment:
this pattern must be a constant and better to use LocalDate instead of Date
https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info#use-localdate-instead-of-date